components/eventing-controller/controllers/subscription/jetstream/reconciler_test.go

```diff
@@ -505,24 +587,53 @@ func TestChangeSubscription(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
+			// The new t instance needs to be used for making assertions. If ignored, errors like this will be printed:
```

Comment:
(nit)

do we need this pretty-long comment in every test ?

Reply:
Good question. I think just by reading the test code it is not obvious why this code exists:

```
			g = gomega.NewGomegaWithT(t)
			ens.G = g
```

What about we remove the second line of the comment ? 

Another idea would be to add more details to the testing guides and then quickly reference it only

Reply:
So far we have 2 occurrences. I think let's leave it as it is until we have the problem in more tests. Then we can still reduce the comments or move it to the testing guidelines. What do you think ? :)

Reply:
2 so far are fine, let's think about it if it gets 2+ :)